app/src/dw/res/xml/preferences_global.xml

```diff
@@ -0,0 +1,404 @@
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
My understanding is that this file completely duplicates the ooniprobe preferences. I think we should start thinking about a way to be sure we always sync up preferences changes. Did you already start to think about this? What do you think is the best strategy to ensure this happens?